tools/autograd/derivatives.yaml

```diff
@@ -2789,6 +2797,22 @@
 - name: _test_warn_in_autograd(Tensor self) -> Tensor
   self: warn_backwards(grad)
 
+- name: _test_autograd_multiple_dispatch.one(Tensor self) -> Tensor
+  dispatch:
+    Default:
+      self: grad.expand(self.sizes()) + 1
+      result: auto_linear
+    AutogradNestedTensor:
+      self: grad.mul(grad)
+    AutogradCUDA:
+      self: grad.expand(self.sizes()) * 2
+
+- name: _test_autograd_multiple_dispatch.two(Tensor self, bool b) -> Tensor
+  dispatch:
+    AutogradNestedTensor:
+      self: grad.mul(grad).add(grad)
+  output_differentiability: [True]
```

Comment:
Is this needed when there is no default?

Reply:
Nope! Will remove